src/pages/home/components/dropdown.module.scss

```
@@ -0,0 +1,54 @@
+@import "../../../assets/stylesheets/colors.scss";
+
+@keyframes growUp {
+  0% {
+    transform: scaleY(1);
+  }
+  80% {
+    transform: scaleY(1.1);
+  }
+  100% {
+    transform: scaleY(0);
+  }
+}
+
+.hideMenu {
+  display: none;
+}
+
+.showMenu {
+  position: absolute;
+  left: 0;
+  right: 0;
+  margin-right: auto;
+  margin-left: auto;
+  display: flex;
+  text-align: center;
+  width: 100px;
+  height: 80px;
+  padding-top: 10px;
+  justify-content: center;
+  background-color: white;
+  border-color: red;
+  border: solid $nav-bar-input-border-focus;
+  border-width: 1px;
```

Please double check, but I think you are stepping on your settings in the color. also, could use a one liner por esto.
```suggestion
  border: 1px solid $nav-bar-input-border-focus;
```